packages/block-editor/src/utils/block-bindings.js

```diff
@@ -83,14 +87,11 @@ export function useBlockBindingsUtils() {
 	 * ```
 	 */
 	const removeAllBlockBindings = () => {
-		const { metadata } = getBlockAttributes( clientId );
-		const newMetadata = { ...metadata };
-		delete newMetadata.bindings;
+		const {
+			metadata: { bindings, ...metadata },
+		} = getBlockAttributes( clientId );
```

Comment:
Not sure if [this comment](https://github.com/WordPress/gutenberg/pull/64740/files#r1731113612) is relevant here as well because I guess `removeAllBlockBindings` should not be called when there are no bindings. But maybe it is better to add a fallback just in case?

Reply:
It doesn't harm to add extra safety.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/64740/commits/9e98a29fc6f9aae646d74361a8d224d7fac291af